### PR TITLE
feat: devnet 3 with evm + fix wss for lower envs

### PR DESCRIPTION
**Pull request by cgilbe27**
Adds devnet 3 with evm + fixes wss for lower envs

### CODE REVIEW

File: testnets/nibirudevnet3/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        {
+          "denom": "nibi",
+          "exponent": 6
+        }
+      ],
+      "base": "unibi",
+      "name": "Nibiru",
+      "display": "nibi",
+      "symbol": "NIBI",
+      "logo_URIs": {
```
Comment (Cordtus) [inline testnets/nibirudevnet3/assetlist.json:21]: I think logo_URIs is sort of soft-deprecated

Comment (cgilbe27) [reply testnets/nibirudevnet3/assetlist.json:21]: Should it just be removed?

Comment (JeremyParish69) [reply testnets/nibirudevnet3/assetlist.json:21]: it's fine to keep for now

File: testnets/nibirudevnet3evm/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "nibirudevnet3evm",
```
Comment (JeremyParish69) [inline testnets/nibirudevnet3evm/assetlist.json:3]: This naming doesn't follow current usage at the Chain Registry. Can we use 'nibirudevnet4' instead (it doesn't matter that it doesn't match the chain_id)?

**Comment by cgilbe27** [comment]
@JeremyParish69 addressed

**Pull request merged**